qubes-rpc/Makefile

```diff
@@ -1,16 +1,75 @@
-CC=gcc
-CFLAGS=-g -O2 -Wall -Wextra -Werror -I. -fPIC -pie
+BINDIR ?= /usr/bin
+LIBDIR ?= /usr/lib
+SYSCONFDIR ?= /etc
+QUBESLIBDIR = $(LIBDIR)/qubes
+QUBESRPCCMDDIR = $(SYSCONFDIR)/qubes-rpc
+QUBESCONFDIR = $(SYSCONFDIR)/qubes
+QUBESRPCCONFDIR = $(QUBESCONFDIR)/rpc-config
+
+CPPFLAGS := -I.
+CFLAGS := $(if $(DEBUG),-g,) -O2 -Wall -Wextra -Werror -fPIC -pie $(CFLAGS)
```

Comment:
I'd prefer to keep `-g` enabled. Most packaging systems require it, to extract debug symbols into separate files/packages.

Reply:
Ok, I kept it enabled by default and disabled only if DEBUG=0 is passed.